<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfile</class>
 <widget class="QDialog" name="EditProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑个人信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size:18px;font-weight:bold;margin:6px 0;</string>
     </property>
     <property name="text">
      <string>个人信息编辑</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName">
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>请输入中文姓名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBirth">
       <property name="text">
        <string>出生日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEditBirth">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelId">
       <property name="text">
        <string>身份证号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditIdNumber">
       <property name="maxLength">
        <number>18</number>
       </property>
       <property name="placeholderText">
        <string>18位，最后一位可为X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPhone">
       <property name="text">
        <string>手机号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPhone">
       <property name="maxLength">
        <number>11</number>
       </property>
       <property name="placeholderText">
        <string>11位手机号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="maxLength">
        <number>254</number>
       </property>
       <property name="placeholderText">
        <string>example@domain.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConfirm">
       <property name="text">
        <string>确认</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>dateEditBirth</tabstop>
  <tabstop>lineEditIdNumber</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>pushButtonBack</tabstop>
  <tabstop>pushButtonConfirm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
